Vagrantfile

```diff
@@ -117,10 +117,17 @@ Vagrant.configure("2") do |config|
 
 	# Ensure that WordPress can install/update plugins, themes and core
 	if vagrant_version >= "1.3.0"
-		config.vm.synced_folder ".", "/vagrant", :mount_options => [ "dmode=777,fmode=777" ]
-		CONF["synced_folders"].each do |from, to|
-			config.vm.synced_folder from, to, :mount_options => [ "dmode=777,fmode=777" ]
-		end if CONF["synced_folders"]
+		if CONF['virtualbox']['nfs']
```

Comment:
I was wondering about that when I coded it up. I decided on VB only because @joehoyle mentioned in Slack he was having issues with NFS on VMWare but I didn't have any specific details.

@joehoyle What are your thoughts Sir? VB only or should this flag enable it for both?